doc/cli-interaction.md

```diff
@@ -0,0 +1,80 @@
+# ARGO Compute Engine CLI
+
+## Executable Scripts for cli interaction with the compute engine
+In the folder `/usr/libexec/ar-compute/bin/` reside executable scripts that can be used for uploading metric data and sync data to the hadoop cluster (HDFS Filesystem).
+
+|Script | Description | Shortcut|
```

Comment:
I have not edited the cli-interaction yet. Will do so after we finish with the other two documents